app/src/main/res/layout/audio_fragment.xml

```diff
@@ -51,18 +56,17 @@
       android:progressDrawable="@drawable/audio_seekbar_background"
       android:splitTrack="false"
       android:thumb="@drawable/seekbar_thumb"
-      android:layout_marginStart="16dp"
       android:thumbOffset="16dp"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toStartOf="@+id/tvAudioLanguage"
+      app:layout_constraintEnd_toStartOf="@+id/ivAudioLanguage"
       app:layout_constraintStart_toEndOf="@+id/ivPlayPauseAudio"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <TextView
-      android:id="@+id/tvAudioLanguage"
+    <ImageView
+      android:id="@+id/ivAudioLanguage"
```

Comment:
This doesn't follow our convention for naming IDs. It should be snakecase, and ideally it shouldn't include the view type in the name (this is a convention we moved away from earlier in the project). Suggest instead: 'audio_language_icon'.

Reply:
@BenHenning may I change all the ids of audio_fragment because all of them are following the format as `ivAudioLanguage`

Reply:
updated ids of `audio_fragment` to new namming convention